utils/DeployedContractsHelpers.ts

```diff
@@ -155,7 +155,7 @@ export function getTreasurySigner(hre: HardhatRuntimeEnvironment): Promise<Signe
 }
 
 export async function getTreasuryAddress(hre: HardhatRuntimeEnvironment): Promise<string> {
-  return hre.network.name === "mainnetFork" ? (await hre.ethers.getNamedSigner("TREASURY")).address : constants.treasuryAddresses[hre.network.name];
+  return hre.network.name === "mainnetFork" ? (await getTreasurySigner(hre)).address : constants.treasuryAddresses[hre.network.name];
```

Comment:
@tkeidar FYI I performed the PR comment here by mistake instead of in the previous PR